### DISCUSSION TITLE
为什么第一次文档加载完内容后，会自动滚动到文档最底部？

### DISCUSSION THREAD

**Discussion by shenghui-shan** [umodoc/editor#15] [Q&A]
文档加载完内容后，会自动滚动到文档最底部，请问怎么禁用这个功能呢？

**Comment by umodoc** [comment]
@shenghui-shan 可通过在文档内容加载完成后调用focus方法解决该问题，如：
```vue
<template>
  <umo-editor ref="editorRef" ... />
</template>
<script setup>
const editorRef = ref()

// 组件初始化完成后设置文档内容
editorRef.value.setContent('很长很长的文档内容...')
editorRef.value.foucs() // 见：https://editor.umodoc.com/cn/docs/methods#focus
</script>
```

**Reply by shenghui-shan** [reply]
@umodoc  好的 感谢！！

**Reply by umodoc** [reply]
> @shenghui-shan 可通过在文档内容加载完成后调用focus方法解决该问题，如：
> 
> ```
> <template>
>   <umo-editor ref="editorRef" ... />
> </template>
> <script setup>
> const editorRef = ref()
> 
> // 组件初始化完成后设置文档内容
> editorRef.value.setContent('很长很长的文档内容...')
> editorRef.value.foucs() // 见：https://editor.umodoc.com/cn/docs/methods#focus
> </script>
> ```

@shenghui-shan 在即将发布的 v2.2.0版本及后续版本，可向`setContent`传递`foucs`的相关参数，控制`focus`的行为，无需在设置文档内容后再调用`focus`方法。
<img width="725" alt="image" src="https://github.com/user-attachments/assets/78459dcc-1db7-4fca-bbb2-38373c7f5f78">

**Reply by umodoc** [reply]
> > @shenghui-shan 可通过在文档内容加载完成后调用focus方法解决该问题，如：
> > 
> > ```
> > <template>
> >   <umo-editor ref="editorRef" ... />
> > </template>
> > <script setup>
> > const editorRef = ref()
> > 
> > // 组件初始化完成后设置文档内容
> > editorRef.value.setContent('很长很长的文档内容...')
> > editorRef.value.foucs() // 见：https://editor.umodoc.com/cn/docs/methods#focus
> > </script>
> > ```
> 
> @shenghui-shan 在即将发布的 v2.2.0版本及后续版本，可向`setContent`传递`foucs`的相关参数，控制`focus`的行为，无需在设置文档内容后再调用`focus`方法。
> <img width="725" alt="image" src="https://github.com/user-attachments/assets/78459dcc-1db7-4fca-bbb2-38373c7f5f78">

@shenghui-shan v2.2.0版本已发布，文档已同步更新
